web/server/main.py

```diff
@@ -27,6 +28,8 @@ async def dispatch() -> None:
     app.state.console_listeners = []
     app.state.dispatch_task = asyncio.create_task(dispatch())
 
+    asyncio.create_task(watch_project(api_console.queue))
```

Comment:
Should we cancel the task here: https://github.com/TobikoData/sqlmesh/blob/main/web/server/main.py#L32